lib/Sema/InstrumenterSupport.h

```diff
@@ -0,0 +1,152 @@
+//===--- InstrumenterSupport.h - Instrumenter Support -----------*- C++ -*-===//
+//
+// This source file is part of the Swift.org open source project
+//
+// Copyright (c) 2014 - 2016 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See https://swift.org/LICENSE.txt for license information
+// See https://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+//
+//===----------------------------------------------------------------------===//
+//
+//  This file implements the supporting functions for writing instrumenters of
+//  the Swift AST.
+//
+//===----------------------------------------------------------------------===//
+
+#include "TypeChecker.h"
+
+using namespace swift;
```

Comment:
Do you need this using declaration if everything in the header is inside that namespace?  You're opting everyone who includes it into swift.